### PR TITLE
227 connect idservice to daisy

### PR DESCRIPTION
**Pull request by jLebioda**

### CODE REVIEW

**Comment by jLebioda** [comment]
What's missing is automatic publishing unpublished Partners/Projects linked to a Dataset - I will add it presently

File: web/templates/projects/project.html [hunk 1]
```diff
@@ -65,6 +65,28 @@ <h1>{{ project.acronym }} </h1>
...
+                        <p class="text-center mt-5">
+                            {% if not project.is_published %}
+                                <a href="{% url 'project_publish' pk=project.pk %}" 
+                                   id="publish" 
+                                   title="Publish project"
+                                   class="btn btn-primary bmd-btn-fab float-right">
+                                    <i class="material-icons">publish</i>
+                                </a>
+                            {% else %}
+                                {% comment %}
```
Comment (vildead) [inline web/templates/projects/project.html:78]: Just to confirm - this doesn't have to be a comment any more, right? The views are just redirecting without actually doing anything :+1:

**Review by vildead — CHANGES_REQUESTED** [review]
Just parking it here. If you add comments, I can probably fix those:

- [ ] 1. do we need the "Is published?" field in entity editor page? (project/cohort/partner)
- [ ]  At the moment, super user can unpublish the entity by unchecking the checkbox :/ Also this field is very misleading for the users.
- [ ] 2. It's possible to edit "elu_accession" attribute for published datasets (Dataset Editor page) :scream: 
- [x] 3. Corner ribbon "Published" does not show up for published project (entity overview box)
- [ ] 4. When publishing dataset, we could include a modal box with notification that also subentities (cohorts, partners, project, ...) will be published.

**Comment by jLebioda** [comment]
Will close this PR; this will be merged in #243

**Pull request closed without merging**